packages/object/src/index.ts

```diff
@@ -203,11 +255,10 @@ export class DRPObject implements IDRPObject {
 	}
 
 	// check if the given peer has write permission
-	private _checkWriterPermission(drp: DRP, peerId: string): boolean {
-		if (drp.acl) {
-			return drp.acl.isWriter(peerId);
-		}
-		return true;
+	private _checkWriterPermission(peerId: string): boolean {
+		return this.acl
+			? Reflect.get(this.acl, "isWriter").call(this.acl, peerId)
```

Comment:
```suggestion
			? (this.acl as IACL).isWriter(peerId)
```
is `Reflect` more performant than this?

Reply:
No, I changed as you suggestion